invenio_cli/helpers/commands.py

```diff
@@ -43,6 +43,25 @@ def __getattr__(self, name):
         """Delegate commands according to environment."""
         return getattr(self.environment, name)
 
+    def destroy(self, local):
```

Comment:
I wonder if there should be a default value for the parameter `local`?

Reply:
In my understanding, the `local` parameter changes according to the environment. What would be the reason for having it defaulted?

Reply:
it says in the definition, `--local` defaults to 'local', where is the defaulting happening? sorry...i didnt check in detail. Also, before you had 2 identical functions, one using `local` and one didn't. That triggered my question that maybe local is optional...

Reply:
No worries! Still thank you for looking into my code :)